Composer/packages/server/src/externalContentProvider/powerVirtualAgentsProvider.ts

```diff
@@ -141,6 +142,10 @@ export class PowerVirtualAgentsProvider extends ExternalContentProvider<PowerVir
     return this.getAccessToken();
   }
 
+  public async generateProfile(): Promise<PublishTarget> {
+    throw new Error('PVA Not support');
+  }
```

Comment:
you have defined this as an optional function inside of `externalContentProvider.ts` so you shouldn't need to implement it for `PowerVirtualAgentsProvider`